quandl/get_pit_data.py

```diff
@@ -0,0 +1,70 @@
+from quandl.model.pit import Pit
+from quandl.errors.quandl_error import LimitExceededError
+from .api_config import ApiConfig
+from .message import Message
+import warnings
+import copy
+
+
+def get_pit_data(datatable_code, **options):
```

Comment:
IMO I think that this would be better named `get_point_in_time_data` because its public facing and gives a better indication to the caller what `pit` means